src/Phinx/Migration/Manager.php

```diff
@@ -134,10 +134,11 @@ public function printStatus($environment, $format = null)
      *
      * @param string    $environment Environment
      * @param \DateTime $dateTime    Date to migrate to
+     * @param boolean   $dryRun      Output SQL rather than executing it
      *
      * @return void
      */
-    public function migrateToDateTime($environment, \DateTime $dateTime)
+    public function migrateToDateTime($environment, \DateTime $dateTime, $dryRun)
```

Comment:
Both [come from](http://docs.phinx.org/en/latest/configuration.html#environments) [environmental configuration](https://github.com/robmorgan/phinx/blob/a601da707f00d40749fe8b2439c28368ed06daed/src/Phinx/Console/Command/Migrate.php#L85-L92). The identifier for the environment is [passed to the Manager](https://github.com/robmorgan/phinx/blob/a601da707f00d40749fe8b2439c28368ed06daed/src/Phinx/Console/Command/Migrate.php#L111-L113) which is [used to retrieve the Environment](https://github.com/robmorgan/phinx/blob/a601da707f00d40749fe8b2439c28368ed06daed/src/Phinx/Migration/Manager.php#L271) that invokes the migration. It's essentially going through the same workflow, except that the identifier for the configuration is what's being passed around, as opposed to the argument received from the console.